src/builtins/core/date.rs

```diff
@@ -180,6 +180,62 @@ impl PartialDate {
 }
 
 /// The native Rust implementation of `Temporal.PlainDate`.
+///
+/// Represents a calendar date like "March 15th, 2024" without any time or timezone
```

Comment:
```suggestion
/// Represents a calendar date like without any time or timezone
```

nit: The date language used in this case could be i18n sensitive formatting wise.